tools/lsp/preview.rs

```diff
@@ -358,60 +358,92 @@ fn evaluate_binding(
     drop_location::workspace_edit_compiles(&document_cache, &edit).then_some(edit)
 }
 
+fn convert_simple_string(input: slint::SharedString) -> String {
+    format!("\"{}\"", input.to_string())
```

Comment:
Don't the string needs to be escaped?

Reply:
Probably yes:-)

Do we have a helper for that?

Reply:
I used `std::escape_debug` as recommended by @ogoffart .